src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -1,46 +1,86 @@
 package cromwell.engine
 
+import java.util.UUID
+
 import akka.actor.{Actor, Props}
 import akka.event.{Logging, LoggingReceive}
 import cromwell.binding._
 import cromwell.binding.values.WdlValue
-import cromwell.engine
-import cromwell.engine.backend.Backend
+import cromwell.engine.backend.{TaskAbortedException, Backend}
 import cromwell.engine.workflow.WorkflowActor
 import cromwell.engine.workflow.WorkflowActor.CallFailed
 
 import scala.language.postfixOps
-import scala.util.{Failure, Success}
-
+import scala.util.{Try, Failure, Success}
 
 object CallActor {
+
   sealed trait CallActorMessage
-  case class Start(inputs: CallInputs) extends CallActorMessage
-  case class ExecutionFinished(outputs: CallOutputs) extends CallActorMessage
-  case class ExecutionFailed(regret: Throwable) extends CallActorMessage
+  case class Start(call: Call, locallyQualifiedInputs: Map[String, WdlValue], backend: Backend, workflowDescriptor: WorkflowDescriptor) extends CallActorMessage
```

Comment:
It's a bit of a 6 of one, half dozen of another situation so I was mainly just curious.

I'd completely agree w/ you though if we were reusing CallActors. In this situation there's a 1-to-1 correspondence between the Call and the CallActor so I'd argue that all of this information is intrinsic to the CallActor type.